source/operations/concepts.rst

```diff
@@ -149,12 +149,14 @@ For more information on write preference calculation logic, see :ref:`Writing Fi
 Rebalancing data across all pools after an expansion is an expensive operation that requires scanning the entire deployment and moving objects between pools.
 This may take a long time to complete depending on the amount of data to move.
 
-Starting with MinIO Client version RELEASE.2022-11-07T23-47-39Z, you can manually initiate a rebalancing operation across all server pools using :mc:`mc admin rebalance`. 
+MinIO does not recommend manual rebalancing.
```

Comment:
Can we remove the recommendation text?

Reply:
@kannappanr you mean we should say it's ok to manually rebalance?

Reply:
Should we instead use a cautionary statement of using only with consultation with MinIO Engineering?
The ask we had for this PR was specifically to discourage use of this feature.

Reply:
Correct we have made a release already, and the fixes are also in the EOS binaries so to some extent we have addressed this already. 

We should perhaps talk about a more broader tone that rebalance is not a real requirement if you size your pools properly.

Reply:
Basically discourage budget setups

Reply:
don't expand in this manner 

first pool 

- 100 nodes which is now 90% used

you botched buying hardware now you just expand by 20 nodes

- 20 nodes is second pool 

This 20 nodes will take all the I/O hit causing significant slowness, the sizing must be appropriate to the load that 100 node was handling. if 20 can handle and its a new hardware no problem but if its not then it is going to cause outage etc.

A cautionary guidance on why rebalance don't solve the problem of high utilization the second pool.  IT may look like that but it won't solve the problem.